packages/cli/src/cli/cogify/action.cog.ts

```diff
@@ -126,6 +128,20 @@ export class ActionCogCreate extends CommandLineAction {
       } else {
         logger.warn('DryRun:Done');
       }
+    } catch (e) {
+      const jobId = Config.ProcessingJob.id(job.id);
+      const jobConfig = await Config.ProcessingJob.get(jobId);
+      if (jobConfig != null) {
+        const jobFailed = jobConfig as ProcessingJobFailed;
+        jobFailed.status = 'failed';
+        if (typeof e === 'string') {
```

Comment:
`jobFailed.error = String(e)`

Reply:
You don't need `instanceof Error` this can be simplified to remove the if statement.